### DISCUSSION TITLE
Sending completed plots over FTP

### DISCUSSION THREAD

**Discussion by Alqueraf** [swar/Swar-Chia-Plot-Manager#168] [Q&A]
Hello,

Any ideas on how to automate sending completed plots over FTP (or something else) to a remote computer?

**Comment by raid5** [comment]
I know a lot of people use `rsync` for transferring plots over a network.

**Reply by Alqueraf** [reply]
I'm doing that for linux/mac machines but couldn't set it up for Windows.
Currently uploading them manually through FTP using WinSCP , which is not ideal

**Reply by goldcd** [reply]
Somebody pointed me towards rclone, which is a nice tool (with a windows version) for shifting stuff about.
Reason I'm using it, is that it lets me limit the bandwidth for moving my plots over the network (as I'm farming off my plotter machine, and saturated network was impacting my farming)
